staging/src/k8s.io/cloud-provider/controllers/node/node_controller_test.go

```diff
@@ -702,6 +702,7 @@ func Test_syncNode(t *testing.T) {
 							Effect: v1.TaintEffectNoSchedule,
 						},
 					},
+					ProviderID: "fake://12345",
```

Comment:
For reference, this was originally removed here https://github.com/kubernetes/kubernetes/pull/95342/files#r501940685 since the fake cloud provider did not support pre-existing provider IDs